docs/UserGuide.md

```diff
@@ -71,31 +71,31 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
-### Adding a person: `add`
+### Adding an interviewee: `add`
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS j/JOB [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01, j/software engineer`
+* `add n/Betsy Crowe e/betsycrowe@example.com a/Street street p/1234567 t/good interviewee j/data analyst`
```

Comment:
Your format is job before tag but the example is tag before job. Not sure it might mess with the parsing or not

Reply:
Thanks for the comment! If I'm not wrong the original ab3 user guide also placed the tag at the front to show that the order doesn't matter. I've tested it and it seems like the order of all the fields can be switched.